### PR TITLE
Add coloring for birthday and tooltip

### PR DESCRIPTION
**Pull request by fymbc**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -121,6 +121,7 @@ public static Tag parseTag(String tag) throws ParseException {
         if (!Tag.isValidTagName(trimmedTag)) {
             throw new ParseException(Tag.MESSAGE_CONSTRAINTS);
         }
+        assert !trimmedTag.isEmpty();
```
Comment (meatyturtle) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:124]: good job using assertions to maintain the defensiveness of the code

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 2]
```diff
@@ -130,8 +131,18 @@ public static Tag parseTag(String tag) throws ParseException {
     public static Set<Tag> parseTags(Collection<String> tags) throws ParseException {
         requireNonNull(tags);
         final Set<Tag> tagSet = new HashSet<>();
+        final Set<String> netWorthTags = Set.of("highnetworth", "midnetworth", "lownetworth");
+        boolean hasNetWorthTag = false;
```
Comment (meatyturtle) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:135]: is this boolean field used anywhere else in the code?

File: src/main/resources/view/DarkTheme.css [hunk 3]
```diff
@@ -350,3 +350,30 @@
     -fx-background-radius: 2;
     -fx-font-size: 11;
 }
+
```
Comment (meatyturtle) [inline src/main/resources/view/DarkTheme.css:353]: it may be useful to have a new css file for this instead of throwing it in dark theme

**Review by meatyturtle — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 4]
```diff
@@ -96,6 +95,12 @@ public CommandResult execute(Model model) throws CommandException {
     /**
      * Creates and returns a {@code Person} with the details of {@code personToEdit}
      * edited with {@code editPersonDescriptor}.
+     *
+     * If the new tags contain a net worth status tag (either "highnetworth", "midnetworth", or "lownetworth"),
+     * any existing net worth status tags on the person will be removed to ensure that only one net worth status tag
```
[the remark this answers is not in the document]
Comment (fymbc) [reply src/main/java/seedu/address/logic/commands/EditCommand.java:100]: Yes! Included in the visual features section

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 5]
```diff
@@ -105,7 +110,19 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
...
+
+        Set<String> netWorthTags = Set.of("highnetworth", "midnetworth", "lownetworth");
+
+        Optional<Tag> newNetWorthTag = updatedTags.stream()
+                .filter(tag -> netWorthTags.contains(tag.tagName.toLowerCase()))
+                .findFirst();
+
+        if (newNetWorthTag.isPresent()) {
+            updatedTags.removeIf(tag -> netWorthTags.contains(tag.tagName.toLowerCase()));
+            updatedTags.add(newNetWorthTag.get());
```
[the remark this answers is not in the document]
Comment (fymbc) [reply src/main/java/seedu/address/logic/commands/EditCommand.java:123]: Yes, will be affected. What do you think?

File: src/main/java/seedu/address/logic/parser/ParserUtil.java (hunk 2 above)
Comment (fymbc) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java:135]: negative, it is meant for looping through to find existing networth tags

File: src/main/resources/view/DarkTheme.css (hunk 3 above)
Comment (fymbc) [reply src/main/resources/view/DarkTheme.css:353]: Ok, let's have a look at this at v1.6

**Pull request merged**